technical-infrastructure/template-jenkinsfile-for-jdk11-maven-projects.md

```diff
@@ -12,37 +12,48 @@ See [Naming Conventions]({% link technical-infrastructure/naming-conventions.md
 
 ## Overview
 
-This shows the general form of a Jenkinsfile used with a Maven project based on JDK11.
+This shows an example of a Jenkinsfile that could be used with a Maven project based on JDK11.
```

Comment:
Is JDK11 still relevant?

Reply:
Yes, it is. CVS still uses JDK 11 and it's a standard baseline within the Java community.